person_class.rb

```diff
@@ -24,4 +26,8 @@ def correct_name
   def of_age?
     return false unless @age >= 18
   end
+
+  def add_rental(date, book)
+    Rental.new(date, self, book)
+  end
```

Comment:
Great job on implementing the `has_many` relationship between  `person` and `rentals`